src/main/java/uk/gov/cabinetoffice/csl/service/IdentityService.java

```diff
@@ -88,12 +111,22 @@ public Identity getIdentityForEmail(String email) {
         return identityRepository.findFirstByEmailEqualsIgnoreCase(email);
     }
 
-    public Identity getIdentityForEmailAndActiveFalse(String email) {
+    public Identity getActiveIdentityForEmail(String email) {
+        return identityRepository.findFirstByActiveTrueAndEmailEqualsIgnoreCase(email);
+    }
+
+    public Identity getInActiveIdentityForEmail(String email) {
```

Comment:
I think `InActive` should be `Inactive`.

Reply:
Done